fhir-bulkdata-webapp/src/main/java/com/ibm/fhir/bulkdata/export/group/resource/GroupHandler.java

```diff
@@ -83,10 +77,20 @@ public void process(String groupId) throws Exception {
         }
     }
 
-    /*
+    /**
+     * get a page of members from this handler
+     *
+     * @param pageNum
+     * @param pageSize
+     * @return
+     * @implNote {@link GroupHandler#process(String)} must be called first
```

Comment:
I think the design of this could be improved, but at this point I just tried making it clear that GroupHander is stateful and that process needs to be called before you can page through the members

Reply:
Yes,  I tried to split  the prior implementation  so it can be further refactored.